qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -4,26 +4,25 @@ Frequently Asked Questions
 What kind of data can I upload to Qiita for processing?
 -------------------------------------------------------
 
-We need 3 things: raw data, sample template, and prep template. At this
-moment, raw data is fastq files without demultiplexing with forward,
-reverse (optional) and barcode reads. We should have before the end of
-the week SFF processing so it's OK to upload. Note that we are accepting
```

Comment:
Remove: We should have before the end of the week SFF processing so it's OK to upload.